Line/android/LINELoginDemo/app/src/main/java/com/google/firebase/linelogindemo/activity/MainActivity.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.linelogindemo.activity;
+
+import android.app.ProgressDialog;
+import android.content.DialogInterface;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.android.volley.toolbox.ImageLoader;
+import com.android.volley.toolbox.NetworkImageView;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.linelogindemo.R;
+import com.google.firebase.linelogindemo.util.LineLoginHelper;
+import com.google.firebase.linelogindemo.util.NetworkSingleton;
+
+public class MainActivity extends AppCompatActivity {
+
+    // Views
+    private Button mLineLoginButton;
+    private View mLoggedInView;
+    private TextView mDisplayNameText;
+    private NetworkImageView mProfileImageView;
+
+    // Helpers
+    private ImageLoader mImageLoader;
+    private LineLoginHelper mLineLoginHelper;
+
+    private final String TAG = this.getClass().getSimpleName();
```

Comment:
Fixed in this [commit](https://github.com/firebase/custom-auth-samples/pull/1/commits/fd56eda6713e7fd016d5c8ad820c78d2ffda97bf)